### ISSUE TITLE
Android build doesn't save settings

### ISSUE THREAD

**Issue by ShadowNinja** [minetest/minetest#2083]
Tested on a Nexus 7 (2012).
Also happened on celeron55's Jolla.

**Comment by sapier** [comment]
hmm doesn't happen on my xperia M ... wonder what's required to make it happen

**Comment by MoNTE48** [comment]
I've seen this bug a year ago, but now everything is working perfectly. You can close it.

**Comment by sapier** [comment]
closing due to no additional error information provided for almost a year

**Comment by paramat** [comment]
Ok.

**Comment by stujones11** [comment]
I am unable to reproduce this on either of my devices. @Wayward1 can you confirm this is still happening with recent builds?

**Comment by paramat** [comment]
@MoNTE48 @juozaspo @IhrFussel @codexp @benrob0329 @srifqi @rubenwardy have you experienced this bug?

**Comment by srifqi** [comment]
No, I never have experienced this bug on my old Android 2.3.6 and Android 6.0.1.

**Comment by srifqi** [comment]
No, I never have experienced this bug on my old Android 2.3.6 (Minetest 0.4.x) and Android 6.0.1 (Minetest 5.0.0).

**Comment by juozaspo** [comment]
On my device I didn't encounter such bug. Settings (minetest.conf) are saving fine on my android builds, though I didn't test latest master yet.
